<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>UpdateTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>配置选项</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>156</height>
      </rect>
     </property>
     <property name="title">
      <string>连接设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>156</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>目标端口:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>8888</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="TestBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>测试连接</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>156</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP地址:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>192.168.1.12</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>TestBtn</zorder>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>181</width>
       <height>226</height>
      </rect>
     </property>
     <property name="title">
      <string>固件信息</string>
     </property>
     <widget class="QPushButton" name="LoadBtn">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>载入...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateBtn">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>升级</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>190</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="fwNamelabel">
         <property name="text">
          <string>固件名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fwSizelabel">
         <property name="text">
          <string>固件大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fwDate_label">
         <property name="text">
          <string>日期：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>616</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>固件数据</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>616</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>提示信息</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_ctrl">
    <property name="title">
     <string>操作(&amp;C)</string>
    </property>
    <addaction name="update_action"/>
    <addaction name="separator"/>
    <addaction name="test_action"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_ctrl"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open_action">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="update_action">
   <property name="text">
    <string>升级(&amp;U)</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="test_action">
   <property name="text">
    <string>测试连接(&amp;T)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
